### PR TITLE
Dra 251 kaltura integration test

### PR DESCRIPTION
**Pull request by thomasegense**
Try run the unittest from IDE and you should be able to produce the API error. 
Remember kb init

### CODE REVIEW

File: src/test/java/dk/kb/image/kaltura/KalturaApiIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import dk.kb.image.config.ServiceConfig;
+import dk.kb.image.util.KalturaUtil;
+
+/**
+ * Unittest that will call the API api search method. Search for a local refenceId to get the Kaltura internal id for the record.
```
Comment (VictorHarbo) [inline src/test/java/dk/kb/image/kaltura/KalturaApiIntegrationTest.java:12]: API api. please delete one

File: src/test/java/dk/kb/image/kaltura/KalturaApiIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        String referenceId="9ad1d1aa-be25-4466-ada7-1e3c1e140e98";
+        String kalturaInternallId="0_g9ys622b";
+ 
+                
+        int success=0;
+        for (int i = 0;i<20;i++) {
+            DsKalturaClient client = KalturaUtil.getClientInstance(); //This will reuse the session as it is configured in KalturaUtil.            
+            
+            String kalturaId = client.getKulturaInternalId(referenceId);
+            assertEquals(kalturaInternallId, kalturaId,"API was reproduced after number of calls:"+success);
```
Comment (VictorHarbo) [inline src/test/java/dk/kb/image/kaltura/KalturaApiIntegrationTest.java:39]: Please update the message. Maybe something like: "API error was reproduced after '" + succes + "' number of calls"

File: src/test/java/dk/kb/image/kaltura/KalturaApiIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            success++;
+            Thread.sleep(1000L);
+                System.out.println(kalturaId);
```
Comment (VictorHarbo) [inline src/test/java/dk/kb/image/kaltura/KalturaApiIntegrationTest.java:42]: Is this println left over? if not please change to a log message

File: conf/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+/ds-image-kaltura-integration-test.yaml
```
Comment (VictorHarbo) [inline conf/.gitignore:1]: Could we maybe add the values in these added git ignores to the overall .gitignore. 
I dont think we need three ignore files here

File: src/test/resources/.gitignore [hunk 5]
```diff
@@ -0,0 +1 @@
+/ds-image-integration-test.yaml
```
Comment (VictorHarbo) [inline src/test/resources/.gitignore:1]: Same as the other gitignore file

**Review by VictorHarbo — CHANGES_REQUESTED** [review]
Minor changes to the setup around the test and structure. 
The test fails on my machine as well

File: src/test/java/dk/kb/image/kaltura/KalturaApiIntegrationTest.java (hunk 1 above)
Comment (thomasegense) [reply src/test/java/dk/kb/image/kaltura/KalturaApiIntegrationTest.java:12]: fixed

File: src/test/java/dk/kb/image/kaltura/KalturaApiIntegrationTest.java (hunk 2 above)
Comment (thomasegense) [reply src/test/java/dk/kb/image/kaltura/KalturaApiIntegrationTest.java:39]: fixed

File: conf/.gitignore (hunk 4 above)
Comment (thomasegense) [reply conf/.gitignore:1]: fixed

File: src/test/resources/.gitignore (hunk 5 above)
Comment (thomasegense) [reply src/test/resources/.gitignore:1]: fixed

**Review by VictorHarbo — APPROVED** [review]
Looks good now

**Pull request merged**